app/src/main/res/layout/fragment_home.xml

```diff
@@ -98,4 +98,9 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/searchAndFilterView" />
 
+    <com.movies.presentation.view.loader.LoaderDialog
+        android:id="@+id/loaderView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"/>
+
```

Comment:
I don't think loaderView should be a part of the xml in fragment_home. it will be better for the view to be attached programatically in base. otherwise you have to include these lines in every fragment xml

Reply:
done